### PR TITLE
Do not exclude mininum starting price for resource if it is zero (TAM-129)

### PR DESCRIPTION
**Pull request by urshala**
When the price for the resource is rendered, currently it excludes
the min price if it is zero. Do not exclude the min price even if
it is zero. If it is zero render it as 0 - max_price in resource info
and resource card.

### CODE REVIEW

File: src/domain/resource/utils.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ export const getPrice = (minPriceString, maxPriceString, priceType, t) => {
     week: t('common.unit.time.week'),
   });
 
-  if (minPrice && maxPrice && minPrice !== maxPrice) {
+  if (minPrice >= 0 && maxPrice && minPrice !== maxPrice) {
```
Comment (tuomas777) [inline src/domain/resource/utils.js:95]: I think this will now include also `minPrice` `null`, which is probably not what we want.

Comment (urshala) [reply src/domain/resource/utils.js:95]: Customer wanted `null minPrice` to be rendered as 0. So, for now we want to include `null` value too.

Comment (tuomas777) [reply src/domain/resource/utils.js:95]: Ah ok. IMHO it is not really nice, as those fields are nullable for a reason, and that is to distinguish between "zero" and "not set". But of course there can be some practical reasons which I'm not aware of, and the customer is the king anyway, so all good here 🙂

**Review by tuomas777 — APPROVED** [review]
LGTM 👍

**Pull request merged**